### PR TITLE
bug fix: hot key view disappear on iOS 15

### PR DESCRIPTION
**Pull request by alex-yatsenko-dev**
Thanks for proposing a pull request!

To help us review the request, please complete the following:

- [x ] sign [contributor license agreement](https://code.facebook.com/cla)
- [x ] I've ensured that all existing tests pass and added tests (when/where necessary)
- [x ] I've updated the documentation (when/where necessary) and [Changelog](CHANGELOG.md) (when/where necessary)
- [x ] I've added the proper label to this pull request (e.g. `bug` for bug fixes)

## Pull Request Details

fixed bug with immediate closing of hot keys view on iOS 15. Fixed issue https://github.com/facebook/facebook-ios-sdk/issues/1959

## Test Plan

Test Plan: start app with inited FB SDK, hold command key on keyboard, hot keys view should appear.

### CODE REVIEW

**Comment by Jeabluckycoin** [comment]
เมื่อ จ. 29 พ.ย. 2564 เวลา 20:36 alex-yatsenko-dev ***@***.***>
เขียนว่า:

**Comment by alex-yatsenko-dev** [comment]
Do I need to take any actions?

**Comment by masumh** [comment]
Thank you

**Comment by masumh** [comment]
alex-yatsenko-dev:hot_keys_view_ios15_fix

**Comment by masumh** [comment]
Please help me automatic adjust my problem

**Review by masumh — COMMENTED** [review]
Please help me my problem automatic adjust

**Comment by masumh** [comment]
Please help me automatic adjust my problem

**Review by A3ew3 — COMMENTED** [review]
alex-yatsenko-dev:hot_keys_view_ios15_fixالقيمة

File: FBSDKCoreKit/FBSDKCoreKit/Internal/FBSDKAppEventsUtility.h [hunk 1]
```diff
@@ -45,6 +45,7 @@ NS_SWIFT_NAME(AppEventsUtility)
                       loggingOverrideAppID:(nullable NSString *)loggingOverrideAppID;
 + (BOOL)validateIdentifier:(nullable NSString *)identifier;
 + (BOOL)isSensitiveUserData:(NSString *)text;
++ (BOOL)isHotKeyEvent:(UIResponder *)uiResponder;
```
[the remark this answers is not in the document]
Comment (JeffersonW13) [reply FBSDKCoreKit/FBSDKCoreKit/Internal/FBSDKAppEventsUtility.h:48]: - [ ] - Hello from your friend. Jeff Welch [@]()5864431725 call me

[the remark this answers is not in the document]
Comment (JeffersonW13) [reply FBSDKCoreKit/FBSDKCoreKit/Internal/FBSDKAppEventsUtility.h:48]: Okay thanks again

[the remark this answers is not in the document]
Comment (JeffersonW13) [reply FBSDKCoreKit/FBSDKCoreKit/Internal/FBSDKAppEventsUtility.h:48]: > > alex-yatsenko-dev:hot_keys_view_ios15_fix
> 
> 

- [ ] - there

[the remark this answers is not in the document]
Comment (JeffersonW13) [reply FBSDKCoreKit/FBSDKCoreKit/Internal/FBSDKAppEventsUtility.h:48]: Fucken take him out of my life

**Review by JeffersonW13 — COMMENTED** [review]
Someone is using my iPhone account for Facebook

**Pull request closed without merging**